editor/blocks/quote/index.js

```diff
@@ -0,0 +1,44 @@
+const Editable = wp.blocks.Editable;
+const { query, html } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'core/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+	category: 'quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: html( 'cite,footer' )
```

Comment:
Probably, the only problem is that this will not work for our example `post-content.js` if we change it to just `cite`.  So we will want to make a decision on the markup.